src/plugins/intel_cpu/src/plugin.h

```diff
@@ -37,6 +37,11 @@ class Engine : public InferenceEngine::IInferencePlugin {
     InferenceEngine::IExecutableNetworkInternal::Ptr ImportNetwork(std::istream& networkModel,
                                                      const std::map<std::string, std::string>& config) override;
 
+    std::vector<std::vector<int>> get_streams_info_table(const int input_streams,
+                                                         const int input_threads,
```

Comment:
I suppose it should not be a plugin method

Reply:
This method is based on CPU abstract information and decoupled with HW platform. 

Putting this unified method in plugin give one more option which one CPU plugin work with different StreamsExecutor for different HW platforms.